java/ql/test/query-tests/security/CWE-297/UnsafeHostnameVerification.java

```diff
@@ -0,0 +1,84 @@
+import javax.net.ssl.HostnameVerifier;
```

Comment:
Tests for experimental queries should live under ` java/ql/test/experimental/query-tests`

Reply:
`UnsafeHostnameVerification.ql` is not added as an experimental query.
Or are you suggesting to move it to experimental?
I'm willing to comply with the requirements for supported queries.

Reply:
Doh right, only the old one is. If you're willing to take the higher standards and longer process, I'm happy to review going for main rather than experimental.

Reply:
Yes I'm willing to take the higher standards :+1: